src/aiortc/codecs/keypointcodec.py

```diff
@@ -12,7 +12,7 @@
 
 SCALE_FACTOR = 256//2
 NUM_KP = 10
-NUM_JACOBIAN_BITS = int(os.environ.get('JACOBIAN_BITS', 6))
+NUM_JACOBIAN_BITS = int(os.environ.get('JACOBIAN_BITS', -1))
```

Comment:
Is 'JACOBIAN_BITS' a list? If not, what does it mean to have -1 for the default value?

Reply:
oh got it, nevermind